model_security_adjust_oaw/models/supplier_stock.py

```diff
@@ -90,21 +90,25 @@ def _get_owners_duplicates(self):
                     owners_duplicates.sudo().write({'owners_duplicates': True})
                 else:
                     owners_duplicates.sudo().write({'owners_duplicates': False})
-
     @api.multi
     def write(self, vals):
-        res = super(SupplierStock, self).write(vals)
-        if 'quantity' in vals:
-            for ps in self:
+        for ps in self:
+            if 'quantity' in vals:
                 ps._get_owners_duplicates()
-        if self.env.user.has_group('model_security_adjust_oaw.group_supplier'):
-            server_actions = self.env['base.action.rule'].sudo().search([
-                ('model', '=', 'supplier.stock'),
-                ('kind', 'in', ('on_write', 'on_create_or_write')),
-                ('active', '=', True)
-            ], order='sequence')
-            for action in server_actions:
-                action.sudo()._process(action, [res.id])
+            ps.check_changes(vals)
```

Comment:
Since the `check_changes` method has been removed, please also remove the line here